src/java/org/apache/cassandra/utils/ByteArrayUtil.java

```diff
@@ -195,6 +195,53 @@ public static void putDouble(byte[] b, int off, double val) {
         putLong(b, off, Double.doubleToLongBits(val));
     }
 
+    /**
+     * An efficient way to write the type {@code bytes} of a long
+     *
+     * @param register - the long value to be written
+     * @param bytes - the number of bytes the register occupies. Valid values are between 1 and 8 inclusive.
+     * @throws IOException
+     */
+    public static void putBytes(byte[] b, int off, long register, int bytes)
```

Comment:
Unit test it works in all 9 cases?

Reply:
This should have been removed entirely, my mistake.